packages/dd-trace/src/plugins/util/promise.js

```diff
@@ -1,15 +1,19 @@
 'use strict'
 
+const {
+  bind
+} = require('../../../../datadog-instrumentations/src/helpers/instrument')
+
 module.exports = {
-  createWrapThen (tracer, config) {
+  createWrapThen () {
```

Comment:
This can be removed completely if `knex` is moved and the promise part of `mongoose` uses the new `wrapThen`.